### ISSUE TITLE
Byfl Branch for LLVM 3.9 does not Configure Correctly

### ISSUE THREAD

**Issue by nmhamster** [lanl/Byfl#17]
(1) Performed the configure as required in the `INSTALL` documentation.

(2) Run `./configure --prefix=/home/projects/ecp-dse/tools/byfl/clang/3.9.0`

(3) Output of configure is:

```
checking for dragonegg.so... no
checking that generated files are newer than configure... done
configure: creating ./config.status
config.status: creating Makefile
config.status: creating include/Makefile
config.status: creating lib/Makefile
config.status: error: cannot find input file: `lib/byfl/Makefile.in'
```

**Comment by spakin** [comment]
Strange.  Byfl might have to be built from a subdirectory.  Try
```bash
autoreconf -f -i -v
mkdir build
cd build
../configure --prefix=/home/projects/ecp-dse/tools/byfl/clang/3.9.0
make
make install
```

**Comment by spakin** [comment]
Which version of the GNU Autotools are you running?  I have
```
$ autoconf --version | head -1
autoconf (GNU Autoconf) 2.69
$ automake --version | head -1
automake (GNU automake) 1.15
```

Do you want to submit a pull request for the `AC_CONFIG_MACRO_DIRS([m4])` line, or should I just paste it in?

**Comment by nmhamster** [comment]
@spakin I am checking these. Basically, I've distilled it down slightly to needing to do the following:

(1) Add `AC_CONFIG_MACRO_DIRS([m4])` into `configure.ac`
(2) Run `libtoolize`
(3) Run `autoreconf -f -i`

This seems to build a working `configure`.

Version wise:

- `libtool` 2.4.6
- `automake` 1.15.0
- `autoconf` 2.69.0

**Comment by spakin** [comment]
Fixed by 270e2090f3f9bc184658df6b542ea70babccedb7.
